### ISSUE TITLE
[Feature] Show metrics during epochs in cv like in ts

### ISSUE THREAD

**Issue by nischa564** [sdsc-bw/DataFactory#27]
- during finetuning of a model user can see train_loss, test_loss, accuracy and time for each epoch
- similar to the dataframe of ts 
- maybe use fastai for training
